source/core/utilities.cc

```diff
@@ -485,3 +530,18 @@ get_dimension(const std::string &file_name)
           "Lethe requires that the user explicitly specify the dimension of the problem within the parameter file. This can be achieved by adding set dimension = 2 or set dimension = 3 within the parameter file"));
     }
 }
+
+
+int
+get_max_number_of_boundary_conditions(const std::string &file_name)
+{
+  int max_number_of_boundary_conditions =
+    get_max_value_of_parameter(file_name, "number");
+
+  AssertThrow(
+    max_number_of_boundary_conditions >= 0,
+    dealii::ExcMessage(
+      "Your parameter file does not contain any indication for the number of boundary conditions for any physics supported by Lethe. Since November 2023, Lethe requires that a \"boundary conditions\" subsection be present with at least \"number=0\" "));
```

Comment:
```suggestion
      "Your parameter file does not contain any indication for the number of boundary conditions for any physics supported by Lethe. Since November 2023, Lethe requires that a \"boundary conditions\" subsection is present with at least \"number=0\" "));
```